docs/src/config/darkCodeTheme.js

```diff
@@ -0,0 +1,127 @@
+/** @type {import('prism-react-renderer').PrismTheme} */
+const darkCodeTheme = {
+  plain: {
+    color: '#E3E3E3',
+    backgroundColor: '#323234',
+  },
+  styles: [
+    {
+      types: ['prolog'],
+      style: {
+        color: '#000080'
+      }
+    },
+    {
+      types: ['comment'],
+      style: {
+        color: '#6A9955'
+      }
+    },
+    {
+      types: ['builtin', 'changed', 'keyword', 'interpolation-punctuation'],
+      style: {
+        color: '#569CD6'
+      }
+    },
+    {
+      types: ['number', 'inserted'],
+      style: {
+        color: '#B5CEA8'
+      }
+    },
+    {
+      types: ['constant'],
+      style: {
+        color: '#646695'
+      }
+    },
+    {
+      types: ['attr-name', 'variable'],
+      style: {
+        color: '#9CDCFE'
+      }
+    },
+    {
+      types: ['deleted', 'string', 'attr-value', 'template-punctuation'],
+      style: {
+        color: '#CE9178'
+      }
+    },
+    {
+      types: ['selector'],
+      style: {
+        color: '#D7BA7D'
+      }
+    },
+    {
+      // Fix tag color
```

Comment:
As we aren't adding comments to each object explaining the usage, let's just remove them.

Reply:
That was a comment from the original theme 👼